src/components/TypeAhead.tsx

```diff
@@ -102,4 +110,61 @@ class TypeAhead extends React.PureComponent<TypeaheadProps, TypeaheadState> {
   }
 }
 
-export default TypeAhead;
+export class CachedTypeAhead extends React.PureComponent<
```

Comment:
named exports are not available from outside by default. 2 options here - make this `TypeAhead.Cached` or export explicitly as a new component. I would prefer the first one.